### ISSUE TITLE
请问后三层的卷积层大小为什么一样

### ISSUE THREAD

**Issue by zhanghongsir** [ViswanathaReddyGajjala/EfficientDet-Pytorch#2]

**Comment by likki02** [comment]
I have used EfficientNet-B0 features for the Bi-FPN module. Since the model is small, considering lower level features wouldn't help much. I have done experiments using those feature map sizes. You can change the code according to different feature map sizes.

**Comment by ViswanathaReddyGajjala** [comment]
I have used EfficientNet-B0 features for the Bi-FPN module. Since the model is small, considering lower level features wouldn't help much. I have done experiments using those feature map sizes. You can change the code according to different feature map sizes.

**Comment by zhanghongsir** [comment]
> I have used EfficientNet-B0 features for the Bi-FPN module. Since the model is small, considering lower level features wouldn't help much. I have done experiments using those feature map sizes. You can change the code according to different feature map sizes.

I also want to ask you a question. I changed the FPN in my network into a bifpn. The training can be done normally. If it is tested, the AP is 0. No object can be detected. Do you know why?

**Comment by ViswanathaReddyGajjala** [comment]
I have done experimentations with the code provided in this repro. I have replaced the FPN block with the BiFPN block and there was an improvement in the accuracy, I think there might be some issue in your training script. Try debugging!

**Comment by zhanghongsir** [comment]
> I have done experimentations with the code provided in this repro. I have replaced the FPN block with the BiFPN block and there was an improvement in the accuracy, I think there might be some issue in your training script. Try debugging!

OK, thank you very much. Can you tell me how much the accuracy has improved

**Comment by ViswanathaReddyGajjala** [comment]
1-2 map. Tested on a tiny dataset.

**Comment by zhanghongsir** [comment]
Thank you. Do you use only one bifpn module

> 1-2 map. Tested on a tiny dataset.



> 1-2 map. Tested on a tiny dataset.

Thank you. Do you use only one bifpn module？

**Comment by ViswanathaReddyGajjala** [comment]
I have tried multiple bifpn modules. Model with one bifpn module worked well for me. Based on the complexity of the dataset you need to chose that.

**Comment by zhanghongsir** [comment]
> I have tried multiple bifpn modules. Model with one bifpn module worked well for me. Based on the complexity of the dataset you need to chose that.

Thank you again for your great help
